### PR TITLE
Final version!

### PR DESCRIPTION
**Pull request by sousalorena**
I think I have finished the DBR introductory chapter! :-)

### CODE REVIEW

File: tableOfContents.md [hunk 1]
```diff
@@ -13,3 +13,6 @@ order: 3
 1. [Preface](/detel-book/chapter/introduction/preface/)
 
 2. [My New Chapter](/detel-book/chapter/myChapter/)
+
+3. [Design-based Research in TEL: an introduction](/detel-book/chapters/Lorena_UAveiro/DBR/)
+4. [Creating Open Educational Resources with the H5P tool](/detel-book/chapters/Lorena_UAveiro/OER/)
```
Comment (BenediktHensen) [inline tableOfContents.md:18]: Could you propose a structure for the table of contents that matches with the one of the current live version? https://github.com/ea-tel/detel-book/blob/master/tableOfContents.md

**Review by BenediktHensen — CHANGES_REQUESTED** [review]

File: _chapters/chapters/Lorena_UAveiro/DBR.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
+---
+layout: page
+title: Design-based Research in TEL: an introduction
+
+hide: true
+permalink: /chapters/Lorena_UAveiro/DBR/
```
[the remark this answers is not in the document]
Comment (fominykh) [reply _chapters/chapters/Lorena_UAveiro/DBR.md:6]: Yes, of course, by content and not by authors.

File: tableOfContents.md (hunk 1 above)
Comment (fominykh) [reply tableOfContents.md:18]: I will make a structure and update both in the TOC and the chapters.

File: _chapters/chapters/Lorena_UAveiro/OER.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+ 
+[13] [H5P website .com](https://h5p.com/) 
+ 
+[14] [H5P website org](https://h5p.org/) 
+ 
+[15] [H5P website: content types and applications](https://h5p.org/content-types-and-applications) 
+ 
+[16] Sousa, L., Pedro, L., Santos, C. (2021). Developing a Prototype of an Open Educational Resource on Research Methods for PhD Candidates in Technology-Enhanced Learning. In: De Laet, T., Klemke, R., Alario-Hoyos, C., Hilliger, I., Ortega-Arranz, A. (eds) Technology-Enhanced Learning for a Free, Safe, and Sustainable World. EC-TEL 2021. Lecture Notes in Computer Science(), vol 12884. Springer, Cham. https://doi.org/10.1007/978-3-030-86436-1_23
+ 
+[17] [H5P website: integrations](https://h5p.org/integrations)
```
[the remark this answers is not in the document]
Comment (fominykh) [reply _chapters/chapters/Lorena_UAveiro/OER.md:146]: We will need to approve the input as is and update the bibliography in the next version, I am afraid.

**Review by fominykh — APPROVED** [review]
There are some improvements to make style-wise, but we can publish the chapter as is now.

File: tableOfContents.md (hunk 1 above)
Comment (fominykh) [reply tableOfContents.md:18]: structure is already there. I will adapt to it.

**Review by fominykh — APPROVED** [review]

**Pull request merged**